Extensions/show_originals.js

```diff
@@ -111,11 +111,8 @@ XKit.extensions.show_originals = new Object({
 			return;
 		}
 
-		// This probably doesn't ever happen
-		if (!XKit.extensions.show_originals) {
-			XKit.extensions.show_originals.added_css = true;
-			XKit.tools.add_css(" .post.is_reblog { display: none; ", "show_originals_on");
-		}
+		XKit.extensions.show_originals.added_css = true;
+		XKit.tools.add_css(" .post.is_reblog { display: none; ", "show_originals_on");
```

Comment:
This looks like it doesn't have the closing brace like `.post.is_reblog { display: none; }`